### PR TITLE
fix: Stop TopSites refreshing before TippyTop init. Closes #3352.

### LINKED ISSUE #3352: TopStories redundently fetches screenshots of default sites on first run
On NEW_TAB_LOAD the top sites are refreshed, but this can happen before TippyTopProvider has finished initialising, and so the default sites are queued for screenshots, resulting in network requests to Wikipedia, Facebook, etc.

### PR DESCRIPTION
**Pull request by AdamHillier**
Fix #3352.

### CODE REVIEW

File: system-addon/test/unit/lib/TopSitesFeed.test.js [hunk 1]
```diff
@@ -247,6 +247,19 @@ describe("Top Sites Feed", () => {
     });
   });
   describe("#refresh", () => {
+    it("should initialise _tippyTopProvider if it's not already initialised", async () => {
+      feed._tippyTopProvider.initialised = false;
```
Comment (pdehaan) [inline system-addon/test/unit/lib/TopSitesFeed.test.js]: Should this be `initialised` or `initialized`?

Comment (rlr) [reply system-addon/test/unit/lib/TopSitesFeed.test.js]: doh! nice catch @pdehaan 👍

Comment (AdamHillier) [reply system-addon/test/unit/lib/TopSitesFeed.test.js]: Ah thank you, English English getting me into trouble...

**Comment by as-pine-proxy** [comment]
[Treeherder: fc0f0c18f9e7c975539fd1ef8f686ba2cfeda3b8](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=fc0f0c18f9e7c975539fd1ef8f686ba2cfeda3b8)

**Pull request merged**